app/managing-users.md

```diff
@@ -55,32 +55,32 @@ Okta allows new users to sign in and use RAVS.
 * If users already have an Okta account for other NHS applications, then their RAVS account will be added to their existing Okta account.
 * If users are new to Okta, they’ll also receive an Okta email with instructions to set up their account. They’ll need to do this first to access RAVS.
 
-![Screen shows user a preview email before they send it](/images/manage-users-check.png)
+## Adding a user to multiple organisations 
 
-## Adding users when their ‘email already exists’ 
+From 16 December 2024, you can add a user to more than 1 organisation, using the same email address for that user. 
 
-RAVS cannot onboard users to multiple organisations with the same email address. Only one email can be linked to one organisation. 
+The user will then be able to use the same email address to access RAVS at each organisation. 
```

Comment:
Is it worth spelling out that as well as the same email address it means they'll use the same Okta account (with the same password) for both organisations? Or is it enough that it’s implied?

```suggestion
The user will then be able to use the same email address and Okta account to access RAVS at each organisation. 
```